pkg/api/api.go

```diff
@@ -253,6 +253,30 @@ func canaryTestFailures(all []sippyprocessingv1.FailingTestResult) []sippyv1.Can
 	return canaryFailures
 }
 
+func minimumJobPassRateByBugzillaComponent(report, prev sippyprocessingv1.TestReport) []sippyv1.MinimumPassRatesByComponent {
+	var result []sippyv1.MinimumPassRatesByComponent
+	for c, failures := range report.JobFailuresByBugzillaComponent {
+		passRate := sippyv1.MinimumPassRatesByComponent{
+			Name: c,
+			PassRates: map[string]sippyv1.PassRate{
+				"latest": {
+					Percentage: failures.JobsFailed[0].FailPercentage,
```

Comment:
hm.  you called this field passrate but you're putting a fail rate into it.

Reply:
how about putting `1-FailurePercentage` in here?  Though I think @deads2k might say that's a lie because we only know when this thing failed, we don't know when it passed.

Reply:
Good catch. Fixed.a

Reply:
optimistic pass rates :-)